api/main.py

```diff
@@ -321,3 +321,24 @@ def mock_s3(filename):
     gunicorn_logger = logging.getLogger('gunicorn.error')
     app.logger.handlers = gunicorn_logger.handlers
     app.logger.setLevel(gunicorn_logger.level)
+
+
+def _store_lnauth_key(lnkey):
+    """Grabs the latest LnAuth entry and stores `lnkey` in the field"""
+    ln = m.LnAuth.query.order_by(desc(m.LnAuth.created_at)).first()
+    ln.key = lnkey
+    db.session.add(ln)
+    db.session.commit()
+
+
+@app.cli.command("store-lnauth-key")
```

Comment:
Isn't it nicer to run "flask lnauth" on the command line than "flask store-lnauth-key"?

I'm all for explicit names, but I would get tired of typing taht every time I log in, and will probably choose *not* to use this, for this very reason...